examples/webgl_materials_wireframe.html

```diff
@@ -150,10 +148,10 @@
 						uniforms: { 'thickness': { value: API.thickness } },
 						vertexShader: document.getElementById( 'vertexShader' ).textContent,
 						fragmentShader: document.getElementById( 'fragmentShader' ).textContent,
-						side: THREE.DoubleSide,
-						glslVersion: THREE.GLSL3
+						side: THREE.DoubleSide
 
 					} );
+					material2.extensions.derivatives = true;
```

Comment:
BTW: This line is not necessary with WebGL 2.

Reply:
This PR can be reverted when the renderer will no longer return a webGL 1 context.